README.md

```diff
@@ -3,7 +3,7 @@ About rfc3986
 
 Home: https://rfc3986.readthedocs.io
 
-Package license: Apache 2.0
+Package license: Apache
```

Comment:
It is Apache 2.0. See https://github.com/sigmavirus24/rfc3986/blob/master/LICENSE#L3